cloud/azure/bin/lib/key_vault.sh

```diff
@@ -3,18 +3,8 @@
 # CHARSET is a regex pattern that matches the acceptable characters to 
 # use when generating a secret value
 readonly CHARSET='A-Za-z0-9!"#$%&'\''()*+,-./:;<=>?@[\]^_`{|}~'
+readonly KEY_VAULT_SECRETS_OFFICER_GUID="b86a8fe4-44ce-4948-aee5-eccb2c155cd7"
```

Comment:
issue (blocking): these two vars are duplicated in `cloud/azure/bin/lib/azure.sh`

Reply:
removed from `cloud/azure/bin/lib/azure.sh`